src/lib/bootloader/systemdboot.rb

```diff
@@ -0,0 +1,304 @@
+# frozen_string_literal: true
+
+require "fileutils"
+require "yast"
+require "bootloader/sysconfig"
+require "bootloader/cpu_mitigations"
+
+Yast.import "Report"
+Yast.import "Arch"
+Yast.import "ProductFeatures"
+Yast.import "BootStorage"
+Yast.import "Stage"
+
+module Bootloader
+  # Represents systemd bootloader with efi target
+  # rubocop:disable Metrics/ClassLength
+  class SystemdBoot < BootloaderBase
+    include Yast::Logger
+    include Yast::I18n
+
+    # @!attribute menue_timeout
+    #   @return [Integer] menue timeout
+    attr_accessor :menue_timeout
+
+    # @!attribute secure_boot
+    #   @return [Boolean] current secure boot setting
+    attr_accessor :secure_boot
+
+    def initialize
+      super
+
+      textdomain "bootloader"
+      self.menue_timeout = Yast::ProductFeatures.GetIntegerFeature("globals", "boot_timeout").to_i
```

Comment:
I think that this part should be in propose method and not in initialize.

Reply:
Yes, I have moved it.